src/sidebar/services/frame-sync.ts

```diff
@@ -621,5 +663,6 @@ export class FrameSyncService {
   destroy() {
     this._portFinder.destroy();
     this._listeners.removeAll();
+    this._anchorCheckTimeouts.map(timeout => clearTimeout(timeout));
```

Comment:
```suggestion
    this._anchorCheckTimeouts.forEach(timeout => clearTimeout(timeout));
```

This makes it clear that it is not a mistake that a) nothing consumes the result and b) the closure has side effects.